ClientApp/src/features/Main/services/getNewsList.ts

```diff
@@ -0,0 +1,29 @@
+import ky from 'ky';
+import { REQUEST_RETRY_COUNT } from '../../../share/constants';
+import { News } from '../reducer/types';
+
+const REQUEST_ITEM_COUNT = 10;
+
+interface NewsRequestData {
+    id: number;
+    title: {
+        ru: string;
+        be: string;
+    },
+    previewImage: string;
+}
+
+export default async function getNewList(): Promise<News[]> {
+    const res = await ky.get(
+        `/api/news/list/${REQUEST_ITEM_COUNT}`,
```

Comment:
I would've created something like endpoints.ts to store all the endpoints. will be much more of them eventually